workspaces/ui/src/entrypoints/development.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {Route, Switch, useParams} from 'react-router-dom';
+import ApiSpecServiceLoader from '../components/loaders/ApiLoader';
+import {Provider as DebugSessionContextProvider, useMockSession} from '../contexts/MockDataContext';
+import {ApiRoutes} from '../routes';
+
+export default function Development(props) {
+
+  const {match} = props;
+  const {sessionId} = useParams();
+
+  const debugSession = useMockSession({
+    sessionId: sessionId,
+    path: match.url
+  });
+
+  return (
+    <DebugSessionContextProvider value={debugSession}>
+      <ApiSpecServiceLoader>
+        <ApiRoutes />
+      </ApiSpecServiceLoader>
```

Comment:
I'm not sure we'll want to wrap all the `AppRoutes` (`ApiRoutes`?.. 🤭) in the `ApiSpecServiceLoader`, as the service that is fetched and the stores applied don't apply to all routes => `TestingDashboard`. As I argued in the `LoaderFactory` comment, I reckon you'll want to define this per page or logical group of pages, but not blanket for all routes.